### ISSUE TITLE
blank lines inserted between multiline `//`-based comment

### ISSUE THREAD

**Issue by SethTisue** [olafurpg/scalafmt#123]
Original:
```scala
   // prime sieve; use instead of the test or stream when you know the
   // upper bound of the primes you will need.
   // en.wikipedia.org/wiki/Sieve_of_Eratosthenes
   // this is the genuine sieve algorithm; see www.cs.hmc.edu/~oneill/papers/Sieve-JFP.pdf
   // for a discussion of impostors
```

Scalafmt:
```scala
   // prime sieve; use instead of the test or stream when you know the

   // upper bound of the primes you will need.

   // en.wikipedia.org/wiki/Sieve_of_Eratosthenes

   // this is the genuine sieve algorithm; see www.cs.hmc.edu/~oneill/papers/Sieve-JFP.pdf

   // for a discussion of impostors
```

I would like the input to look like this: like the original

Using
* version: 0.1.2
* style: default

**Comment by SethTisue** [comment]
awesome, thanks for the prompt fixes

**Comment by olafurpg** [comment]
Just released 0.1.3, it should appear on Maven after a few minutes. Thank you for reporting the issues!
